generators/src/new-test-model/new-test-model.generator.ts

```diff
@@ -176,6 +176,7 @@ export const newTestModelGenerator: CodeGenerator = {
             isPresetExampleRequired,
             modelName: `${modelName}Draft`,
             modelCodename: `${modelCodename}-draft`,
+            nonDraftModelName: modelName,
```

Comment:
The solution looks good to me.

I would only suggest to rename this property to `baseModelName`.